cli/info.go

```diff
@@ -56,60 +56,60 @@ func (v *InfoCommand) runInfo() error {
 }
 
 func prettyPrintInfo(cli *Cli, info *types.SystemInfo) error {
-	fmt.Fprintln(os.Stdout, "Containers: ", info.Containers)
-	fmt.Fprintln(os.Stdout, " Running: ", info.ContainersRunning)
-	fmt.Fprintln(os.Stdout, " Paused: ", info.ContainersPaused)
-	fmt.Fprintln(os.Stdout, " Stopped: ", info.ContainersStopped)
-	fmt.Fprintln(os.Stdout, "Images: ", info.Images)
-	fmt.Fprintln(os.Stdout, "ID: ", info.ID)
-	fmt.Fprintln(os.Stdout, "Name: ", info.Name)
-	fmt.Fprintln(os.Stdout, "Server Version: ", info.ServerVersion)
-	fmt.Fprintln(os.Stdout, "Storage Driver: ", info.Driver)
-	fmt.Fprintln(os.Stdout, "Driver Status: ", info.DriverStatus)
-	fmt.Fprintln(os.Stdout, "Logging Driver: ", info.LoggingDriver)
-	fmt.Fprintln(os.Stdout, "Volume Drivers: ", info.VolumeDrivers)
-	fmt.Fprintln(os.Stdout, "Cgroup Driver: ", info.CgroupDriver)
-	fmt.Fprintln(os.Stdout, "Default Runtime: ", info.DefaultRuntime)
+	fmt.Fprintf(os.Stdout, "Containers: %d\n", info.Containers)
```

Comment:
`fmt.Fprintln()` insert spaces between operands automatically. In other words, `fmt.Fprintln(f, "",  a)` prints `" a\n"`, while `fmt.Printf(f, "%d\n", a)` prints `"a\n"`.

Reply:
The reason I prefer `fmt.Fprintf()`, rather than `fmt.Fprintln()`, is that `"Containers:"` and `info.Containers` are actually not of the same kind, so using string formatting here would be more intuitive.